keylime/ca_util.py

```diff
@@ -63,15 +63,22 @@ def load_cert_by_path(cert_path: str) -> Certificate:
     return cert
 
 
-def setpassword(pw: Optional[str]) -> None:
+def setpassword(key_store_pw: Optional[str]) -> None:
     global global_password
-    if not pw:
-        pw = getpass.getpass("Please enter the password to decrypt your keystore: ")
+    if not key_store_pw:
```

Comment:
Does make sense to have an empty password?, if so the condition should be `if key_store_pw is None:`

Reply:
No empty password possible. This code was basically transplanted from the `verifier` to here, in order to keep the code (and messages) as close as original as possible.

Reply:
Fixed